src/object/schema_system_catalog_install_query_spec.cpp

```diff
@@ -1172,6 +1172,7 @@ sm_define_view_stored_procedure_spec (void)
 	    "END AS [return_type], "
 	  "[sp].[arg_count] AS [arg_count], "
 	  "CASE [sp].[lang] WHEN 0 THEN 'PLCSQL' WHEN 1 THEN 'JAVA' ELSE 'UNKNOWN' END AS [lang], "
+          "CASE [sp].[directive] & 1 WHEN 0 THEN 'OWNER' ELSE 'CALLER' END AS [authid], "
```

Comment:
Considering Oracle compatibility, DEFINER and CURRENT_USER seem more suitable.

Reply:
Good point. thank you